Assets/Creatures/api/Genera.cs

```diff
@@ -0,0 +1,9 @@
+using UnityEngine;
+
+namespace Creatures.Api
+{
+    public interface Genera
+    {
+        string GeneraType { get; }
+    }
+}
```

Comment:
I wonder if the standard is to have no newline at the end of a file? hm

Reply:
fixt